hnn_core/gui/gui.py

```diff
@@ -1041,59 +1080,66 @@ def add_connectivity_tab(net, connectivity_out,
     with connectivity_out:
         display(cell_connectivity)
 
+    return net
+
+
+def add_drive_tab(params, drives_out, drive_widgets, drive_boxes, tstop,
+                  layout):
+    net = jones_2009_model(params)
```

Comment:
What's the purpose of making a `net` here? It's a little confusing to instantiate a `Network` just to  access the `._params` attribute.

Reply:
It's used to extract `drive_specs` as we need `net.cell_types`

Reply:
Ah I see, I'm assuming this is only called once when the tab is created?

I only raise it because the `jones_2009_model()` function does a lot, and it seems unnecessary to call it multiple times when you really just want one of the attributes. 

Other consideration is that this hard codes the cell types, rather than pulling them from the network actually being simulated (which may change in the future).

Reply:
Either way it's not a huge deal for now, especially if it isn't clear how to make the change cleanly so happy to merge as is